.github/workflows/native-build-development.yml

```diff
@@ -6,14 +6,16 @@ on:
   schedule:
     - cron: '0 2 * * *'
   repository_dispatch:
+  workflow_dispatch:
 
 jobs:
   build_with_native:
     runs-on: ubuntu-latest
+    if: "github.repository == 'quarkusio/quarkus-quickstarts' || github.event_name == 'workflow_dispatch'"
     steps:
       - uses: actions/checkout@v2
         with:
-          ref: development
+          ref: ${{ github.event_name == 'workflow_dispatch' && github.event.ref || 'development' }}
```

Comment:
Note: This is a "fake ternary" as a workaround for missing native support by GH.